Sources/Elementary/HtmlDocument.swift

```diff
@@ -37,5 +39,10 @@ public extension HTMLDocument {
             }
             Elementary.body { self.body }
         }
+        .attributes(.lang(lang), when: lang != "")
     }
 }
+
+public extension HTMLDocument {
+    var lang: String { "" }
+}
```

Comment:
not a fan of the `""` as magic value, an optional would be better I think...

Reply:
I had it as an optional first, but then realized that if somebody declared `var lang: String = "something"` or just `var lang = "something"` in the HTMLDocument, the value isn't picked up – and the only way to realize this is to check the generated output.

Reply:
I am not sure I follow...

Reply:
ah, I see... oh... hmm... that is not good....
let me think about this a bit.
maybe it is time for a "Language" struct to type the value?